test/skimNtuple_HHbtag.cpp

```diff
@@ -5039,6 +5071,12 @@ int main (int argc, char** argv)
 			  ++theSmallTree.m_njets ;
 			} // loop over jets
 
+		  // only apply HEM1516 veto to specific runs in data + corresponding fraction of MC
+		  if((!isMC and theBigTree.RunNumber < 319077) or
```

Comment:
I guess the RunNumbers alone are unique so that we don't have to check whether we're in 2018 or even B,C,D?

Reply:
yes they are :)